server/vertx/src/main/scala/sttp/tapir/server/vertx/VertxEndpointOptions.scala

```diff
@@ -0,0 +1,47 @@
+package sttp.tapir.server.vertx
+
+import java.io.File
+
+import io.vertx.core.logging.{Logger, LoggerFactory}
+import sttp.tapir.server.{DecodeFailureHandler, LogRequestHandling, ServerDefaults}
+
+import scala.concurrent.ExecutionContext
+
+case class VertxEndpointOptions(
```

Comment:
shouldn't this be `VertxServerOptions` for consistency?

Reply:
In fact, Vert.x already has the notion of `HttpServerOptions`  which acts at a higher level (port, http version, ssl, etc.)
Here, the user has the opportunity (through implicits) to specify options for this very specific endpoint. Which could be really useful, especially for the ExecutionContext, which is very important for Vert.x users.

I can rename it, no problem at all, just not sure whether it's a good move or not.

Reply:
Ah, if vertx has the same thing, this makes sense :) The reason for naming `ServerOptions` naming in other interpreters comes from the fact that this applies when interpreting an endpoint as a server.

We can stick with `VertxEndpointOptions`, or use `VertxEndpointServerOptions`?

Reply:
I'd rather stick with `VertxEndpointOptions`, they're bound to the endpoint, and interpreted when the endpoint is "reached" by an http request (especially the execution context => it's used at runtime, when an http server reaches the endpoint).